pkg/i2gw/notifications/notifications.go

```diff
@@ -73,9 +73,10 @@ func (na *NotificationAggregator) CreateNotificationTables() []*tablewriter.Tabl
 			t.Append(row)
 		}
 
-		tables = append(tables, t)
+		tableString.WriteString(fmt.Sprintf("Notifications from %v\n", provider))
```

Comment:
why this is needed?

Reply:
Earlier i could add the name of provider as a seperate header in the table above the headings of columns, but in tablewriter we cant do that, so to specify the provider for which the table is created i added this line

Reply:
Oh ok. So it looks like just the title of the table? Maybe wrap the provider with quotes or make it capital letters.